### ISSUE TITLE
Images don't show in .md notes

### ISSUE THREAD

**Issue by quesada** [pfalcon/zim-desktop-wiki#12]
After getting the index to show images for new md files, they seem to render without images.
What I've tested...

1. The images are in the folder. 
2. The link to the image is correct
3. dragging and dropping the image will render it

It just doesn't render them 'off the bat'

**Comment by pfalcon** [comment]
To expliticize and summarize what you found in #11, this fork in general is not intended to work with a pre-existing .md motes collection. Instead, its purpose is to start and maintain a Zim notebook, but in any general-purpose format, instead of Zim's proprietary format (so the notebook could be at least viewed with other tools).

So, images are kinda intended to be where Zim puts them, and that's what I tested when I initially hacked up Markdown support.

Beyond that, I actually don't really use images in my personal notebook, out of the concern of the uncontrollable growth of its size, while I'd like to keep mine easily syncable to mobile devices (in constant lack of space and with expensive/slow connectivity).

Bottom line: without a from-scratch repro case, I can't say much beyond the above. (But I should mention that I won't be able to look into such a repro case any time soon.)
